releasenotes/notes/bug-fix-in-mps-unitary-835d77b381480fcd.yaml

```diff
@@ -0,0 +1,8 @@
+---
+
+fixes:
+  - |
+    ``unitary`` gate in MPS did not support permutations in the ordering of the 
+    qubits on which the gate is applied. Added support for permutations. 
+    This bug appeared when using ``fusion`` on MPS.
```

Comment:
It's better to remove the last line. It doesn't give useful information.

Reply:
ok, removed.